src/config.js

```diff
@@ -1,8 +1,8 @@
-import { PAYER_SK, OWNER_SK, DEVELOP_DOMAIN, ADMIN_PASSWORD } from './developmode'
-import winston from 'winston'
-import fs from 'fs'
+import { PAYER_SK, OWNER_SK, DEVELOP_DOMAIN, ADMIN_PASSWORD } from './developmode';
```

Comment:
looks like prettier is somehow creating issues with the current eslint config, might be a good idea to use the npm package https://github.com/hirosystems/eslint-config for eslint